File: kotlin/fizzbuzz.kt
fun main() {
  for (i in 1..100) {
    when {
      (i.mod(3) == 0 && i.mod(5) == 0) -> println("FizzBuzz")
      (i.mod(3) == 0) -> println("Fizz")
      (i.mod(5) == 0) -> println("Buzz")
      else -> println(i)
    }
  }
}

File: elixir/fizzbuzz.ex
defmodule FizzBuzz do
  def fizzbuzz(n) when rem(n, 3) == 0 and rem(n, 5) == 0, do: "FizzBuzz"
  def fizzbuzz(n) when rem(n, 3) == 0, do: "Fizz"
  def fizzbuzz(n) when rem(n, 5) == 0, do: "Buzz"
  def fizzbuzz(n), do: n
end

1..100
|> Enum.map(&FizzBuzz.fizzbuzz/1)
|> Enum.each(&IO.puts/1)
